docs/cluster_up_down.md

```diff
@@ -232,6 +250,40 @@ Once the machine has been created, the `--create-machine` argument is no longer
   C:\> oc cluster down --docker-machine=openshift
   ```
 
+## Persistent Volumes
+
+(Available starting origin release 1.5.0-alpha.2)
+Cluster up creates a set of persistent volumes by default. It exposes a new 
+flag that allows setting the directory on the host for these volumes (`--host-pv-dir`). The registry and templates for 
+databases and jenkins will now default to persistent storage.
+
+## Using a Proxy
+
+**Cluster up flags**
+
+> Available starting origin release 1.5.0-alpha.2
+
+Cluster up supports the following flags to specify a proxy:
+- `--http-proxy` set to URL of HTTP proxy to use
+- `--https-proxy` set to URL of HTTPS proxy to use
+- `--no-proxy` specify a comma-separated list of hosts to not proxy. Cluster up automatically adds a set of hosts to this list such
+   as the current hostname and the docker registry IP
+
+**Docker Settings**
+
+Your Docker settings should mirror the cluster up settings for HTTP and HTTPS proxy. And if you do specify either of these,
+You should at least add the registry service IP (172.30.1.1) to the Docker daemon's NO_PROXY environment variable.
+How these settings are specified will vary depending on your platform. Both Docker for Windows and Docker for Mac provide a 
+settings GUI for specifying a proxy. On RHEL or Fedora, you can add HTTP_PROXY, HTTPS_PROXY, and NO_PROXY variables to 
+/etc/sysconfig/docker (or /etc/sysconfig/docker-latest depending on which service you're using).
+Cluster up will warn you if there is a discrepancy between the Docker settings and the arguments you specify for cluster up.
+
+
+| WARNING |
+| ------- |
+| On Docker for Windows, if you specify proxy settings for Docker in its GUI, Docker will apply those settings to every container that is run on the daemon. This requires that you specifically exclude any services that could be accessed container to container (172.30.0.0/16) |
```

Comment:
@bparees added instructions on how to exclude addresses in Windows.
(btw, I'd completely forgotten about this pull)